docs/api/command-line-switches.md

```diff
@@ -291,6 +291,15 @@ Print stack traces for deprecations.
 
 Print stack traces for process warnings (including deprecations).
 
+### `--dns-result-order=order`
+
+Set the default value of verbatim in `dns.lookup()` and `dnsPromises.lookup()`. The value could be:
+
+* ipv4first: sets default verbatim false.
+* verbatim: sets default verbatim true.
+
+The default is verbatim and `dns.setDefaultResultOrder()` have higher priority than `--dns-result-order`.
+
```

Comment:
Added the links and formatting back from the Node.js docs and changed the wording a bit for clarity:

```suggestion
### `--dns-result-order=order`

Set the default value of the `verbatim` parameter in the Node.js [`dns.lookup()`](https://nodejs.org/api/dns.html#dnslookuphostname-options-callback) and [`dnsPromises.lookup()`](https://nodejs.org/api/dns.html#dnspromiseslookuphostname-options) functions.

The value could be:

* `ipv4first`: sets default `verbatim` `false`.
* `verbatim`: sets default `verbatim` `true`.

The default is `verbatim` and `dns.setDefaultResultOrder()` have higher priority than `--dns-result-order`.

```